runtime/onert/core/src/compiler/ExecutorFactory.cc

```diff
@@ -157,6 +157,9 @@ createBackendContexts(compiler::ILoweredGraph &lgraph, bool linear_executor,
     data.graph = std::move(graph);
   };
 
+  // Append builtin context
+  init_context_data(compiler::BackendManager::get().getBuiltin());
```

Comment:
Previously, the builtin backend was always present in `def_backends`, so its context was implicitly created, but it's wrong. In these `PermutationEliminationPass `changes, there are cases where `def_backends` no longer includes the builtin backend. We need to explicitly initialize and append the builtin backend context to ensure it always exists.